docs/apache-airflow-providers-openlineage/guides/developer.rst

```diff
@@ -50,4 +49,25 @@ Instead of returning complete OpenLineage event, the provider defines ``Operator
       run_facets: dict[str, BaseFacet] = Factory(dict)
       job_facets: dict[str, BaseFacet] = Factory(dict)
 
-OpenLineage integration takes care to enrich it with things like general Airflow facets, proper event time and type.
+OpenLineage integration itself takes care to enrich it with things like general Airflow facets, proper event time and type, creating proper OpenLineage RunEvent.
+
+How to properly implement OpenLineage methods?
+==============================================
+
+There are several things worth noting when implementing OpenLineage in operators.
```

Comment:
```suggestion
There are a couple of things worth noting when implementing OpenLineage in operators.
```

I was anticipating a third thing when reading this, but there seems to be only two?

Reply:
Hmm, I was referring to everything below that line. I guess that's unclear, tried to fix this.